### ISSUE TITLE
GHC 8.8.1 should depend on libtinfo-dev

### ISSUE THREAD

**Issue by kolmodin** [haskell-CI/haskell-ci#304]
I installed GHC 8.8.1 using the instructions here https://downloads.haskell.org/~debian/.

```shell
$ lsb_release -a
No LSB modules are available.
Distributor ID: Debian
Description:    Debian GNU/Linux 9.8 (stretch)
Release:        9.8
Codename:       stretch
```

I then tried to compile haddock which failed with this error message;

```shell
$ cabal new-build
Build profile: -w ghc-8.8.1 -O1
In order, the following will be built (use -v for more details):
 - haddock-api-2.23.0 (lib) (first run)
 - haddock-2.23.0 (exe:haddock) (first run)
Preprocessing library for haddock-api-2.23.0..
Building library for haddock-api-2.23.0..
/usr/bin/ld.gold: error: cannot find -ltinfo
collect2: error: ld returned 1 exit status
`gcc' failed in phase `Linker'. (Exit code: 1)
cabal: Failed to build haddock-api-2.23.0 (which is required by exe:haddock
from haddock-2.23.0).
```

After digging a bit I found that ghc-8.8.1 ships with terminfo which depends on libtinfo-dev. But since GHC doesn't depend on libtinfo-dev the install fails.

Manually installing libtinfo-dev fixes the problem.

It seems reasonable to me that ghc-8.8.1 should depend on libtinfo-dev to avoid this problem.

**Comment by hvr** [comment]
@kolmodin this has been addressed in the revision "16" builds (currently available for Ubuntu Eoan packages; but will be included in the upcoming builds for the Debian 10 Buster packages)

**Comment by hvr** [comment]
This has been now also been addressed in the GHC 8.8.2 builds for Debian 9 & Debian 10
